runhouse/servers/obj_store.py

```diff
@@ -157,6 +162,7 @@ async def ainitialize(
         setup_cluster_servlet: ClusterServletSetupOption = ClusterServletSetupOption.GET_OR_CREATE,
         runtime_env: Optional[Dict] = None,
         logs_level: str = DEFAULT_LOG_LEVEL,
+        enable_telemetry_collection: bool = None,
```

Comment:
also set to true here?

Reply:
i feel like im missing part the logic flow - to avoid confusion in the http server we should be setting its final value, and everywhere else it should be passed in as a boolean